pubspec.yaml

```diff
@@ -120,6 +120,10 @@ flutter:
     - assets/images/guardians/guardian_tutorial_4.png
     - assets/images/guardians/invite_guardian_mail.svg
     - assets/images/guardians/guardian_shield.png
+    - assets/images/guardians/onboarding/onboarding_1.jpg
```

Comment:
are you sure we dont have these files in the app already ?

Reply:
Yes but hey are png, I read that jpg load faster but png have better quality. With one should I keep?.

Reply:
ok keep jpg